docs/rook.md

```diff
@@ -46,173 +45,193 @@ kubectl rook-ceph rook version
-#     {
-#       "lastHeartbeatTime": "2022-09-14T08:57:02Z",
-#       "lastTransitionTime": "2022-09-14T08:56:59Z",
-#       "message": "Cluster created successfully",
-#       "reason": "ClusterCreated",
-#       "status": "True",
-#       "type": "Ready"
-#     },
-#     {
-#       "lastHeartbeatTime": "2022-09-14T08:57:37Z",
-#       "lastTransitionTime": "2022-09-14T08:57:37Z",
-#       "message": "Configuring Ceph Mons",
-#       "reason": "ClusterProgressing",
-#       "status": "True",
-#       "type": "Progressing"
-#     }
-#   ],
-#   "message": "Configuring Ceph Mons",
-#   "observedGeneration": 2,
-#   "phase": "Progressing",
-#   "state": "Creating",
-#   "version": {
-#     "image": "quay.io/ceph/ceph:v17",
-#     "version": "17.2.3-0"
-#   }
-# }
-# cephfilesystemmirrors.ceph.rook.io:
-# cephfilesystems.ceph.rook.io:
-# cephfilesystemsubvolumegroups.ceph.rook.io:
-# cephnfses.ceph.rook.io:
-# cephobjectrealms.ceph.rook.io:
-# cephobjectstores.ceph.rook.io:
-# cephobjectstoreusers.ceph.rook.io:
-# cephobjectzonegroups.ceph.rook.io:
-# cephobjectzones.ceph.rook.io:
-# cephrbdmirrors.ceph.rook.io:
-# objectbucketclaims.objectbucket.io:
+# Info: cephclusters my-cluster
+# ceph:
+#   capacity:
+#     bytesAvailable: 20942770176
+#     bytesTotal: 20971520000
+#     bytesUsed: 28749824
+#     lastUpdated: "2024-01-11T06:44:28Z"
+#   details:
+#     POOL_APP_NOT_ENABLED:
+#       message: 1 pool(s) do not have an application enabled
+#       severity: HEALTH_WARN
+#   fsid: 5a8dc97a-1d21-4c0e-aac2-fdc924e18374
+#   health: HEALTH_WARN
+#   lastChanged: "2024-01-11T06:44:28Z"
+#   lastChecked: "2024-01-11T06:44:28Z"
+#   previousHealth: HEALTH_OK
+#   versions:
+#     mds:
+#       ceph version 18.2.1 (7fe91d5d5842e04be3b4f514d6dd990c54b29c76) reef (stable): 2
+#     mgr:
+#       ceph version 18.2.1 (7fe91d5d5842e04be3b4f514d6dd990c54b29c76) reef (stable): 1
+#     mon:
+#       ceph version 18.2.1 (7fe91d5d5842e04be3b4f514d6dd990c54b29c76) reef (stable): 1
+#     osd:
+#       ceph version 18.2.1 (7fe91d5d5842e04be3b4f514d6dd990c54b29c76) reef (stable): 1
+#     overall:
+#       ceph version 18.2.1 (7fe91d5d5842e04be3b4f514d6dd990c54b29c76) reef (stable): 5
+# conditions:
+# - lastHeartbeatTime: "2024-01-11T05:37:46Z"
+#   lastTransitionTime: "2024-01-11T05:37:46Z"
+#   message: Processing OSD 0 on node "minikube"
+#   reason: ClusterProgressing
+#   status: "True"
+#   type: Progressing
+# - lastHeartbeatTime: "2024-01-11T06:44:28Z"
+#   lastTransitionTime: "2024-01-11T05:38:27Z"
+#   message: Cluster created successfully
+#   reason: ClusterCreated
+#   status: "True"
+#   type: Ready
+# message: Cluster created successfully
+# observedGeneration: 2
+# phase: Ready
+# state: Created
+# storage:
+#   deviceClasses:
+#   - name: hdd
+#   osd:
+#     storeType:
+#       bluestore: 1
+# version:
+#   image: quay.io/ceph/ceph:v18
+#   version: 18.2.1-0
+
+# Warning: resource cephblockpoolradosnamespaces was not found on the cluster
```

Comment:
These warnings are all Info now, need to update the example

Reply:
missed, updating this earlier